layouts/partials/feedback.html

```diff
@@ -4,13 +4,60 @@
 <br>
   <p><b>Was this page helpful?</b></p>
 
-        <a class="btn btn-primary mr-3 mb-4">
+        <a class="btn btn-primary mr-3 mb-4 feedback-yes">
            Yes <i class="fas"></i>
         </a>
-        <a class="btn btn-primary mr-3 mb-4">
+        <a class="btn btn-primary mr-3 mb-4 feedback-no">
            No <i class="fas"></i>
         </a>
         <a class="btn btn-outline-primary mr-3 mb-4" href="https://github.com/cobalthq/cobalt-product-public-docs/issues/new">
                 Create an Issue <i class="fab"></i>
         </a>
-</div>
\ No newline at end of file
+</div>
+<script>
+   const yesButton = document.querySelector('.feedback-yes');
+   const noButton = document.querySelector('.feedback-no');
+
+   const disableButtons = () => {
+      yesButton.classList.add('disabled')
+      noButton.classList.add('disabled')
+   };
+
+   function thanksFeedback(button){
+      button.blur();
+      buttonText = button.innerText
+      button.innerText = 'Thanks for the feedback!'
+      setTimeout(function () {
+         button.innerText = buttonText;
+      }, 2000);
+   }
+
+   const sendFeedback = (value) => {
+      if (typeof dataLayer !== 'object' || typeof dataLayer.push !== 'function') return;
+
+      const args = {
+         event: 'Feedback Buttons',
+         action: 'click',
+         label: window.location.pathname,
+         value: value
+      };
+
+      dataLayer.push(args);
+   };
+
+   if (yesButton !== null ) {
+      yesButton.addEventListener('click', () => {
+         thanksFeedback(yesButton);
+         disableButtons();
+         sendFeedback(1);
+      });
+   }
+
+   if (noButton !== null ) {
+      noButton.addEventListener('click', () => {
+         thanksFeedback(noButton);
+         disableButtons();
+         sendFeedback(0);
+      });
+   }
```

Comment:
Is there a scenario in which the buttons won't be on the screen (aside from another code change removing/altering them)? If they were missing we'd anyways have a bug, since the buttons won't work. 
```suggestion
      yesButton.addEventListener('click', () => {
         thanksFeedback(yesButton);
         disableButtons();
         sendFeedback(1);
      });

      noButton.addEventListener('click', () => {
         thanksFeedback(noButton);
         disableButtons();
         sendFeedback(0);
      });
```

Reply:
I assume this was for any weird case were the buttons were changed. It made more sense when the script and partial were separated.